<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabooEdit</class>
 <widget class="QWidget" name="TabooEdit">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="tlPalabra">
       <property name="text">
        <string>Palabra</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lePalabra"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tlTaboo1">
       <property name="text">
        <string>Taboo 1:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leTaboo1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="tlTaboo2">
       <property name="text">
        <string>Taboo 2:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leTaboo2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="tlTaboo3">
       <property name="text">
        <string>Taboo 3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leTaboo3"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="tlTaboo4">
       <property name="text">
        <string>Taboo 4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leTaboo4"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="tlTaboo5">
       <property name="text">
        <string>Taboo 5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="leTaboo5"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbGuardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
